src/pages/overview/pdf-extract-api/howtos/extract-api.md

```diff
@@ -137,6 +138,10 @@ schema](/extractJSONOutputSchema2.json)):
 | File contains unsupported font                         | BAD_PDF_UNSUPPORTED_FONT                                                                                                                                                                                | The input file contains font data that is corrupted or not supported.                |
 | Large PDF               | BAD_PDF_LARGE_FILE                                                                                                                                                                                | The input file size exceeds the maximum allowed.                 |
 | Protected PDF     | PROTECTED_PDF                                                                   | Unable to extract content. File is password protected.                 |
+| Disqualified Experiment    | DISQUALIFIED_EXPERIMENT                                                | File belongs to an experiment.                    |
```

Comment:
What is an Experiment in terms of Extract? This makes no sense and feels like something internal we are exposing.

Reply:
Yes it was for internal users. Removed the code.